.github/workflows/ci-vulnerability-scans.yml

```diff
@@ -104,8 +129,17 @@ jobs:
           fi
 
       - name: Run Dockle container linter
-        uses: hands-lab/dockle-action@v1
+        uses: erzz/dockle-action@v1.3.1
         with:
           image: ${{ steps.build-image.outputs.image }}
           exit-code: '1'
-          exit-level: WARN
+          failure-threshold: WARN
+          accept-filenames: ${{ env.DOCKLE_ACCEPT_FILES }}
+
+      - name: Save output to workflow summary
+        if: always() # Runs even if there is a failure
+        run: |
+          echo "# Dockle Results" >> $GITHUB_STEP_SUMMARY
+          if [ -f dockle-report.json ]; then
+            cat dockle-report.json >> $GITHUB_STEP_SUMMARY
```

Comment:
if we're going to be outputting json anyways, how about we add backticks to enable syntax highlighting? like:
```suggestion
            echo "```json" >> $GITHUB_STEP_SUMMARY
            cat dockle-report.json >> $GITHUB_STEP_SUMMARY
            echo "```" >> $GITHUB_STEP_SUMMARY
```

Reply:
I added that in this next commit to see what it looks like, I never thought about using markdown